CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# Contributing
+
+:warning: This project is very alpha, expect bugs!
+These instructions are intended for contributors to get a productive workflow when developing the plugin.
+
+This project follows [scalameta's contribution guidelines].
+Please read them for information about how to create good bug reports and submit pull requests.
+
+## Features
+
+- [x] Lint with Scalafix on compile
+- [x] Formatting with Scalafmt
+- [x] Go to definition in project sources using Semanticdb
+- [ ] Go to definition in dependencies
+- [ ] Red squigglies as you type
+- [ ] Auto completions
+- [ ] Signature helper (show parameter list)
+- [ ] Show type at position
+- [ ] Find references
+- [ ] Show docstring
```

Comment:
What do you think about moving features section to the readme? It looks like a roadmap (btw, it could link to #2) and makes people interested in the project

Reply:
I agree with this 👍

Reply:
Done.